phoenicis-javafx/src/main/resources/com/playonlinux/javafx/views/common/defaultTheme.css

```diff
@@ -122,6 +122,14 @@
     -fx-max-height: 2.5em;
 }
 
+#menuPane > .tab-header-area .tab:selected {
```

Comment:
Looks like this:
![gradient](https://cloud.githubusercontent.com/assets/3973260/22175770/2ad283f6-dffc-11e6-8119-740546ac4a84.png)